### PR TITLE
Trello-155 Enable preprod deployment

### PR DESCRIPTION
**Pull request by mahalrmoj**
https://trello.com/c/xHVqOzpw/155-notify-appointment-outcome-via-event-listener

**What does this pull request do?**
Enable preprod deployment

**What is the intent behind these changes?**
Deployment to higher envs

### CODE REVIEW

**Review by tomsmyers — APPROVED** [review]
👍

File: .circleci/config.yml [hunk 1]
```diff
@@ -67,18 +67,18 @@ workflows:
...
+      - request-preprod-approval:
+          type: approval
+          requires:
+            - deploy_dev
+      - hmpps/deploy_env:
+          name: deploy_preprod
+          env: "preprod"
+          context:
+            - hmpps-common-vars
+            - hmpps-delius-interventions-event-listener-preprod
```
Comment (sldblog) [inline .circleci/config.yml:79]: _not a blocker_ This was slightly bothering me with ui and service -- this context is completely identical with `hmpps-interventions-ui-preprod` and `hmpps-interventions-service-preprod`

Would be good to move contexts on a namespace level, not app level at some time

**Review by sldblog — APPROVED** [review]

File: .circleci/config.yml [hunk 2]
```diff
@@ -67,18 +67,18 @@ workflows:
...
+          requires:
+            - request-preprod-approval
```
Comment (sldblog) [inline .circleci/config.yml:81]: Can we drop this and go straight to preprod if dev deploy is successful? 😁

Comment (mahalrmoj) [reply .circleci/config.yml:81]: I'm always a bit reluctant to go straight from dev to preprod, as it removes the opportunity to do a verification test in a fully integrated env before deploying to a higher env.

Comment (sldblog) [reply .circleci/config.yml:81]: I view this is a mechanism to improve our confidence right off the CI/CD pipeline. Whatever we would _do_ on the dev environment to bump our confidence over 95% that it "works", we should automate by default

Also, 99% of our changes are safe and delaying them (for me, it's basically remembering I have a blocked deploy) isn't worth it for the 1%, I think

**Pull request merged**